scripts/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e -u
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+SRCDIR="${DIR}/.."
+
+# Make sure the output dir exists
+OUTDIR=${SRCDIR}/builds
+mkdir -p $OUTDIR
+
+# Generate the asset name
+ASSETNAME=$(node ${DIR}/assetName.js)
+#echo "Output asset to be named: $ASSETNAME"
+
+# Create the tempdir
+TMPDIR=`mktemp -d`
+# echo $TMPDIR
+
+# Copy the source to the tmpdir
+cp -a ${SRCDIR} ${TMPDIR}
+
+# Remove any old node_modules to make sure they get updated
+rm -rf ${TMPDIR}/asset/node_modules
+
+# Build and install asset dependencies
+cd ${TMPDIR}/asset
+yarn --prod --silent --no-progress
+
+# Zip up generated asset directory
+cd ..
+zip -q -r $OUTDIR/$ASSETNAME asset
+ls $OUTDIR/$ASSETNAME
+zipinfo $OUTDIR/$ASSETNAME
```

Comment:
I like to have some diagnostic output in the CI logs, but the size of this output it pretty excessive.  I'm debating commenting this out.